Random errors on Chrome in Circle-CI

Some tests seem to fail, but only sometimes, on Circle-CI with the Selenium chromedriver, with the following:

```

    def readinto(self, b):
        """Read up to len(b) bytes into the writable buffer *b* and return
            the number of bytes read.  If the socket is non-blocking and no bytes
            are available, None is returned.
    
            If *b* is non-empty, a 0 return value indicates that the connection
            was shutdown at the other end.
            """
        self._checkClosed()
        self._checkReadable()
        if self._timeout_occurred:
            raise OSError("cannot read from timed out object")
        while True:
            try:
>               return self._sock.recv_into(b)
E               ConnectionResetError: [Errno 104] Connection reset by peer

/usr/local/lib/python3.6/socket.py:586: ConnectionResetError
```

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -32,7 +32,7 @@ jobs:
             tar zxf geckodriver-v0.21.0-linux64.tar.gz -C firefox
 
             # Get recent version of chromedriver
-            wget https://chromedriver.storage.googleapis.com/2.40/chromedriver_linux64.zip
+            wget https://chromedriver.storage.googleapis.com/2.41/chromedriver_linux64.zip
             unzip chromedriver_linux64.zip
             mv chromedriver firefox
 

